mne/io/edf/edf.py

```diff
@@ -233,14 +246,18 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
                     stim[n_start:n_stop] += evid
                 data[stim_channel_idx, :] = stim[start:stop]
             else:
-                # Allows support for up to 17-bit trigger values (2 ** 17 - 1)
                 stim = np.bitwise_and(data[stim_channel_idx].astype(int),
-                                      131071)
+                                      2**17 - 1)
+
+                if biosemi:
+                    stim = stim >> 8  # not sure why but it works
```

Comment:
Hmm... that seems like more of a bug than a feature to me.

I'm still +1 on making documentation about it instead of adding an option -- it's more conservative from an API perspective because we can always easily add some option `event_shift` or so later if we realize it's really helpful, whereas removing the `biosemi` arg or making it more general somehow is not so easy. So how about for now we don't do the shift, and if users (yourself included) find it too cumbersome we add something later?